apps/backend/src/drink-action/dto/drink-action.dto.ts

```diff
@@ -1,8 +1,10 @@
+import { OmitType } from '@nestjs/swagger';
+
 import { Drink } from '../../drinks/entities/drink.entity';
 import { Event } from '../../events/entities/event.entity';
 import { DrinkAction } from '../entities/drink-action.entity';
 
-export class DrinkActionDto extends DrinkAction {
+export class DrinkActionDto extends OmitType(DrinkAction, ['eventId']) {
   drink: Drink;
   event: Event;
 }
```

Comment:
You shouldn't edit this file. This DTO represents the type that the GET endpoints of the drink action controller return.

You need to create a new DTO that is very similar to the CreateDrinkActionDto, but with the eventId (you can use OmitType, like here)